index.js

```diff
@@ -38,6 +40,11 @@ function loadAndValidateEnvironment (fastify, opts, done) {
   if (!Array.isArray(opts.data)) {
     data = [data]
   }
+
+  if (opts.dotenv) {
+    require('dotenv').config(Object.assign({}, opts.dotenvConfig))
```

Comment:
why `Object.assign`?

Reply:
... no idea. I think it's an artifact from when I was toying with a merging the user's config options with a set of default options. 
I'll remove